doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/editing-data-sets-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='editing-data-sets-proc_{context}']
+= Editing data sets
+
+You can edit existing data sets to ensure that the data fetched to the reporting components is up-to-date.
+
+.Procedure
+. In {CENTRAL}, go to *Admin* -> *Data Sets*.
++
+The *Data Set Explorer* page opens.
```

Comment:
This is not exactly true. The page that opens is Data Set Explorer + Data Set Authoring Home/Data Set Explorer. Preferably, exact naming should be used. In this case, I would say "Data Sets" page opens, for example.

Reply:
You are correct and I have changed it to the "Data Sets" page.